src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -1233,8 +1251,7 @@ public Task InvokeAsync(object[] parameters)
             }
         }
 
-        // Represents all the transient state about a connection
-        // This includes binding information because return type binding depends upon _pendingCalls
+        //TODO: Refactor all HubConnection into the ConnectionState class.
```

Comment:
Nit:

```suggestion
        //TODO: Refactor all transient state about a connection into the ConnectionState class.
```

Reply:
I don't know if I'd even bother with the TODO. We don't *know* it's worth doing until we see issues that indicate the refactor would solve a problem.

This is getting super nit-picky though so 🤷‍♂️

Reply:
I'll just leave it for now with the update so that moving forward it can at least serve as a sort of reminder that new state should be added to the `ConnectionState `object